### ISSUE TITLE
Configuration Management, part 2: Section 3, LDP compliance requirement too broad

### ISSUE THREAD

**Issue by DavidJHoney** [oslc-op/oslc-specs#525]
Configuration Management 1.0 Part 2: Versioned Resources
Section 3 Version Resources

> Version resources should be compliant with [LDP]. [config-vr-4]

Is this too broad to be meaningful?
For example, some ELM applications do not support JSON-LD (`application/ld+json`) but may support most other parts.
OSLC Core already states:
> OSLC Services should provide and accept RDF documents in Turtle format (identified by the MIME-type 'text/turtle') and in JSON-LD format (identified by the MIME-type 'application/ld+json') representations for each OSLC resource for compatibility with LDP 1.0. [core-7]

[Linked Data Platform 1.0](https://www.w3.org/TR/ldp/) has this:
> 4.2.4.3 If an otherwise valid HTTP PUT request is received that attempts to change properties the server does not allow clients to modify, LDP servers must fail the request by responding with a 4xx range status code (typically 409 Conflict). LDP servers should provide a corresponding response body containing information about which properties could not be persisted. The format of the 4xx response body is not constrained by LDP.

Yet elsewhere in OSLC, there is no requirement that non-persisted or attempted read-only property changes are reported as an error. 

> Version resources should be compliant with [LDP]. [config-vr-4]
Since it's a **SHOULD** and not a **MUST** perhaps this doesn't matter?

**Comment by jamsden** [comment]
This looks like three different issues related to differences between OSLC and LDP:

1. Servers should be compliant with LDP - do you mean SHOULD, or is this informative. I'm ok with SHOULD, but OSLC and LDP aren't really that connected in the spec or in practice. And config management only uses LDP as an implementation detail. 

2. Servers should support JSON-LD - LDP and OSLC are not consistent on JSON-LD, we discussed this in the past and the current core specification on RDF resource representations should be followed by configuration management, not LDP.

3. attempting to write read-only properties - follow OSLC Core guidelines.

Basically when we shifted OSLC Core 3.0 from LDP focused to compatible with OSLC 2.0, we put compatibility head of LDP. The intent was to preserve and promote integrations, and not have technology become an inhibitor.

**Comment by DavidJHoney** [comment]
Let's discuss this next meeting.

**Comment by DavidJHoney** [comment]
In the OSLC OP meeting 2021-07-22, Jim agreed that since the issues I raised were SHOULD not MUST, there wasn't a compliance issue for the configuration management spec. So closing this issue and leaving the spec unchanged.
